core/src/main/java/com/bumble/appyx/core/integration/NodeHost.kt

```diff
@@ -28,11 +29,7 @@ fun <N : Node> NodeHost(
     integrationPoint: IntegrationPoint,
     factory: NodeFactory<N>
 ) {
-    val node by rememberNode(factory, customisations)
-    DisposableEffect(integrationPoint) {
-        integrationPoint.attach(node)
-        onDispose { integrationPoint.detach() }
-    }
+    val node by rememberNode(factory, customisations, integrationPoint)
```

Comment:
`NodeCustomisationDirectory` is neither Stable nor Immutable, does it affects us?
Also if I do not pass `customisations` we create new `NodeCustomisationDirectoryImpl` on each composition.
(can be addressed as a separate PR)

Reply:
good point, we can remember it to prevent unnecessary recompositions